libs/coin-modules/coin-xrp/src/api/index.ts

```diff
@@ -32,8 +33,13 @@ export function createApi(config: XrpConfig): Api<XrpAsset> {
   };
 }
 
+export type TransactionIntentExtra = {
+  destinationTag?: number | null | undefined;
```

Comment:
```suggestion
  destinationTag?: number;
```
Why allowing `null` and `undefined` ? Is the type above not enought ?

Reply:
It's to match the type of the lower level API, I did not want to change it: https://github.com/LedgerHQ/ledger-live/blob/develop/libs/coin-modules/coin-xrp/src/logic/craftTransaction.ts#L48